### PR TITLE
Interactive list prompt

### PR DESCRIPTION
**Pull request by florimondmanca**
Following up on #7:

- Use [Bullet](https://github.com/Mckinsey666/bullet) on *nix:

screenshot 2019-02-27 at 21 59 45

- Fallback to a custom prompt on Windows (user can enter either the number or the actual option text)

screenshot 2019-02-27 at 21 56 22

### CODE REVIEW

**Comment by florimondmanca** [comment]
One caveat is that, for some reason, Bullet seems to not listen to `Ctrl+C` events. This means the user can't cancel while in a list prompt. I looked through the code/docs for Bullet and there doesn't seem to be an option for that. 😞

File: setup.py [hunk 1]
```diff
@@ -29,6 +29,11 @@ def get_version() -> str:
     return str(ast.literal_eval(version))
 
 
+DEPENDENCIES = ["jinja2<2.20, >2.0"]
+
+if not sys.platform.startswith("win"):
```
Comment (cs01) [inline setup.py]: It looks like the canonical way to do this is with `platform_system `. https://stackoverflow.com/a/32643122/2893090

**Comment by cs01** [comment]
Thanks! Looks nice, I'll test it out as soon as I can.

File: setup.py (hunk 1 above)
Comment (florimondmanca) [reply setup.py]: Just learnt something new! 🙌 I pushed a fix.

**Comment by cs01** [comment]
Great work! Thanks so much for doing this :fireworks:

**Pull request merged**